txlib/api/tests/utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+from txlib.registry import registry
+from txlib.tests.compat import MagicMock
+
+
+def setup_registry(get=None, post=None, put=None, delete=None):
+    """Add a mock http_handler in the registry, with custom return values.
+
+    Supports all CRUD operations.
+
+    :param str get: the json string to return for all GET requests
+    :param str post: the json string to return for all POST requests
+    :param str put: the json string to return for all PUT requests
+    :param str delete: the json string to return for all DELETE requests
+    """
+    handler = MagicMock()
+    registry.setup({'http_handler': handler})
+
+    if get is not None:
+        registry.http_handler.get.return_value = json.loads(get)
```

Comment:
Why do you use strings as payloads for mocking and the your parse them as JSON? I think we could work directly with dictionaries in the test files.